aws/eks/main.tf

```diff
@@ -26,7 +26,7 @@ resource "aws_iam_role_policy_attachment" "ssm" {
 module "eks" {
   source                        = "terraform-aws-modules/eks/aws"
   create_eks                    = var.create_eks
-  version                       = "~> 11"
```

Comment:
I've read the changelog and we should be fine. Anyway, we should test that this doesn't change our terraform plan after merging

Reply:
I tested it out and it just wanted to add 2 additional IAM permissions so we should be ok